ins-node/hardware/README.md

```diff
@@ -22,9 +22,9 @@ INS-node board schematics and layout designed with Eagle CAD.
 
 ## List of materials
 * Wemos D1 Mini
-* ATtiny25
+* ATtiny85
 * 2 x AAA battery holders
```

Comment:
I thought we are planning for 3 x AAA batteries now. or?

Reply:
Indeed we do! 👍 Will fix.